.ruby-version

```diff
@@ -1 +1 @@
-2.7
+3.0
```

Comment:
Could we go to 3.1?

Reply:
I wish, but the infrastructure pieces don't have it installed just yet (but I have a [PR](https://github.com/18F/identity-base-image/pull/230))